app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -59,9 +59,15 @@ window.GOVUK.Modules.GovukAccordion = window.GOVUKFrontend.Accordion;
 
   // Navigate to and open accordions with anchored content on page load if a hash is present
   GemAccordion.prototype.openByAnchorOnLoad = function () {
+    if (!window.location.hash) return
     var splitHash = window.location.hash.split('#')[1]
+    var targetID = window.location.hash
```

Comment:
Tiny change suggestion here but it looks like 'ID' tends to be written as 'Id' so perhaps we should update the variable name to `targetId`